test/StressTests/Voron/HugeTransactions.cs

```diff
@@ -131,7 +131,7 @@ private static unsafe void VerifyData
             Assert.Equal(desired, val);
         }
 
-        [Theory64Bit(Skip = "RavenDB-11725")]
+        [Theory64Bit]
```

Comment:
What is the difference?

Reply:
at v4.2 it passes.  at 4.1 it fails.  I am searching the problem for 4.1 right now.. Please wait before merging..